source/agora/consensus/ValidatorSet.d

```diff
@@ -40,6 +40,10 @@ public class ValidatorSet
     /// SQLite db instance
     private Database db;
 
+    /// The available round for re-enrollment that is configurable
+    static private const uint ReenrollAvailableRound =
```

Comment:
`static private const uint ReEnrollAvailableRound =`

How about this?

Reply:
```suggestion
    static private const uint MinimumValidateBlocks =
```

Maybe something like this. The Validator must validate at least this amount of blocks before the enrollment can be replaced.

I am not sure how well this fits in with the whitepaper though.

@Geod24? Would this break any assumptions about how enrollments work?

Reply:
The name I suggested isn't perfect. Feel free to come up with something better if you can, or just leave it as-is.

Reply:
> I am not sure how well this fits in with the whitepaper though.
> 
> @Geod24? Would this break any assumptions about how enrollments work?
> 

Your question is reasonable. I just wanna some constraint against re-enrolling so often. The idea about that is open.